tests/compile_upp.sh

```diff
@@ -6,20 +6,51 @@
 
 set -eu
 
+if [[ $(uname -s) == Darwin ]]; then
+  readonly MYDIR=$(cd "$(dirname "$(greadlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+else
+  readonly MYDIR=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+fi
+PATHTR=${PATHTR:-$( cd ${MYDIR}/.. && pwd )}
+source ${PATHTR}/tests/detect_machine.sh
+
+set_defaults() {
+    delete_exec=YES
+    upp_name="upp.x"
+    load_ifi_module=NO
+    prefix="../install"
+    ifi_opt=" -DBUILD_WITH_IFI=OFF"
+    build_ifi_executables_opt=" "
+    build_ifi_executables=NO
+    gtg_opt=" -DBUILD_WITH_GTG=OFF"
+    nemsio_opt=" -DBUILD_WITH_NEMSIO=ON"
+    wrfio_opt=" -DBUILD_WITH_WRFIO=ON"
+    more=" "
+    verbose_opt=""
+    debug_opt=""
+
+    if [[ "${MACHINE_ID}" == ursa ]] ; then
+        compiler="intelllvm"
```

Comment:
I think I would prefer that we stick with intel for now on Ursa.  I know it produces warnings, but this is in line with other repositories, which all point to the classic compilers by default, including on Ursa.

Reply:
Done.